src/components/category-box/category-box-styles.module.scss

```diff
@@ -48,11 +48,25 @@
   margin: 0;
   margin-bottom: 8px;
   user-select: none;
+  display: flex;
+  align-items: baseline;
+}
+
+.counter {
+  align-items: center;
+  background-color: $robins-egg-blue;
```

Comment:
I see that this is the colour previously used but I would suggest to use `$brand-color-main` (that is the `$java` colour instead of `$robins-egg-blue`).
Also the project colour variable `$dark-text` instead of `$firefly`.
That way if we apply platform wide colour changes on the projects variables those would be applied.
```scss
background-color: $brand-color-main;
color: $dark-text;
```

Reply:
@weberjavi thanks for catching that, ready again